src/s2geography/distance_test.cc

```diff
@@ -0,0 +1,16 @@
+
+#include <gtest/gtest.h>
+
+#include "s2geography.h"
+
+using namespace s2geography;
+
+TEST(DistanceTest, DistanceTestDistance) {
```

Comment:
```suggestion
TEST(Distance, PointDistance) {
```

("Test" probably unnecessary to include for specific test case; could probably be omitted from test suite name too, though examples in documentation show it with test in the name)